gremlin-python/src/main/jython/gremlin_python/structure/io/graphsonV3d0.py

```diff
@@ -513,17 +548,83 @@ class Int64IO(_NumberIO):
     graphson_type = "g:Int64"
     graphson_base_type = "Int64"
 
+    @classmethod
+    def dictify(cls, n, writer):
+        # if we exceed Java long range then we need a BigInteger
```

Comment:
Shouldn't we throw in case we exceed the range of a int64?

Reply:
in python `long` is not limited to 64 bits so i thought it would be a better translation to magically convert to `BigInteger` so that java could consume it.  not good?

Reply:
That is correct. Note that this serializer function is used by the BigIntegerIO as well. So otherwise.. we would have no way to write a `gx:BigInteger`.